### PR TITLE
Fixed issue where third-party providers would not work

### PR DESCRIPTION
**Pull request by Matei9721**
Changed webID field to storage location field which in turn lets us create/update/delete files on third-party pod providers.

### CODE REVIEW

File: app/svelte/src/lib/components/Profile.svelte [hunk 1]
```diff
@@ -7,7 +9,7 @@
 
 <div
 	class="rounded-full bg-green-600 text-green-100 h-12 w-12 flex items-center cursor-pointer"
-	on:click
+	on:click={async ()=> await tester()}
```
Comment (Denperidge) [inline app/svelte/src/lib/components/Profile.svelte:12]: Is there a reason tester() gets called here? Because its mostly meant for, well, testing

**Review by Denperidge — APPROVED** [review]
The comment is about something that won't stop it from working, so consider it approved from me!

**Comment by Denperidge** [comment]
Does this fix #53 btw?

File: app/svelte/src/lib/components/Profile.svelte (hunk 1 above)
Comment (Matei9721) [reply app/svelte/src/lib/components/Profile.svelte:12]: @Denperidge  Yeah, I will remove it. I forgot about it, it was for showing to Thor and Abel that it works.

**Comment by Matei9721** [comment]
> Does this fix #53 btw?

Yes it does!

File: app/svelte/src/lib/components/Profile.svelte (hunk 1 above)
Comment (Denperidge) [reply app/svelte/src/lib/components/Profile.svelte]: No problem, makes sense! Glad to see the tester function has use!

**Pull request merged**